hack/components/bump-kubemacpool.sh

```diff
@@ -83,7 +86,10 @@ rm -rf data/kubemacpool/*
     ./build/_output/bin/go/bin/kustomize build config/cnao | \
         sed 's/kubemacpool-system/{{ .Namespace }}/' | \
         sed 's/RANGE_START: .*/RANGE_START: {{ .RangeStart }}/' | \
-        sed 's/RANGE_END: .*/RANGE_END: {{ .RangeEnd }}/'
+        sed 's/RANGE_END: .*/RANGE_END: {{ .RangeEnd }}/' | \
+        sed 's/AFFINITY/{{ toYaml .Placement.Affinity | nindent 8 }}/' | \
```

Comment:
IIUC you're only seding this in order to remove the `''` right?

Reply:
Yes, expanding the structure with surrounding quotes leads to invalid yaml